media-libs/imlib2/imlib2-1.4.9.recipe

```diff
@@ -0,0 +1,120 @@
+SUMMARY="An advanced replacement library for libraries like libXpm"
+DESCRIPTION="A library that does image file loading and saving as well as \
+rendering, manipulation, arbitrary polygon support, etc.
+
+It does ALL of these operations FAST. Imlib2 also tries to be highly \
+intelligent about doing them, so writing naive programs can be done easily, \
+without sacrificing speed.
+
+This is a complete rewrite over the Imlib 1.x series. The architecture is more \
+modular, simple, and flexible. See index.html in the doc/ directory for more \
+information."
+HOMEPAGE="https://www.enlightenment.org/"
+COPYRIGHT="2002 Project Enlightenment"
+LICENSE="IMLIB2"
+SOURCE_URI="http://downloads.sourceforge.net/enlightenment/imlib2-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7d2864972801823ce44ca8d5584a67a88f0e54e2bf47fa8cf4a514317b4f0021"
+REVISION="1"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	imlib2$secondaryArchSuffix = $portVersion  compat >= 1
+	lib:libimlib2$secondaryArchSuffix = 1.4.9 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libid3tag$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	imlib2${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libimlib2$secondaryArchSuffix = 1.4.9 compat >= 1
+	cmd:imlib2_config$secondaryArchSuffix = $portVersion
+	cmd:imlib2_conv$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	imlib2$secondaryArchSuffix == $portVersion base
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libid3tag$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
```

Comment:
It might be a good idea to add a **`>= 5`** to force imlib2 to depend on libtiff 5.2.4 from tiff4-4.0.6.recipe (instead of libtiff 3.9.7 from tiff-3.9.7.recipe).
```
	devel:libtiff$secondaryArchSuffix >= 5
```